sdk/proto/events/event.proto

```diff
@@ -66,6 +66,10 @@ message ActivityEvent {
 
 // Represents a security violation that is emitted by the agent
 message SecurityViolationEvent {
+  // DEPRECATED
+  reserved 39;
+  reserved "NginxID";
```

Comment:
question to @oliveromahony / @dhurley and the rest of the agent team - how do we add an RN note for this?

Reply:
@mohamed-gougam, no need to wait on this to be resolved for the PR. You can follow up with a documentation update or an RN update. I'll resolve this thread for now, but let's figure this out as a follow on.